src/utils/management/commands/send_email.py

```diff
@@ -0,0 +1,273 @@
+        )
+        parser.add_argument(
+            '--bcc',
+            help='Filepath to JSON file containing a list of email '
+                 'addresses of BCC recipients, relative to Janeway BASE_DIR',
+        )
+        parser.add_argument(
+            '--subject',
+            required=True,
+            help='Subject line'
+        )
+        parser.add_argument(
+            '--body',
+            required=True,
+            help='Filepath to UTF-8 text file, with optional HTML markup, '
+                 'relative to Janeway BASE_DIR',
+        )
+        parser.add_argument(
+            '--confirm',
+            help='Confirm with stdout and command line input before sending. '
+                 'Pass --no-confirm to send immediately',
+            default=True,
+            action=BooleanOptionalAction,
+        )
+        parser.add_argument(
+            '--batchsize',
+            help='Batch size for many BCC recipients',
+            type=int,
+            default=50,
+        )
+
+    def handle(self, *args, **options):
+
+        verbosity = options.get('verbosity', 1)
+
+        journal_code = options.get('journal')
+        if journal_code:
+            try:
+                journal = journal_models.Journal.objects.get(
+                    code=journal_code
+                )
+            except journal_models.Journal.DoesNotExist:
+                logger.error(
+                    self.style.ERROR(
+                        f'Journal code not recognised: {journal_code}'
+                    )
+                )
+                if verbosity >= 1:
+                    self.print_help('manage.py', 'send_email')
+                return
+        else:
+            journal = None
+
+        repository_short_name = options.get('repository')
+        if repository_short_name:
+            try:
+                repository= repository_models.Repository.objects.get(
+                    short_name=repository_short_name
+                )
+            except repository_models.Repository.DoesNotExist:
+                logger.error(
+                    self.style.ERROR(
+                        f'Repository short name not recognised: '
+                        f'{repository_short_name}'
+                    )
+                )
+                if verbosity >= 1:
+                    self.print_help('manage.py', 'send_email')
+                return
+        else:
+            repository = None
+
+        press = press_models.Press.objects.first()
+
+        request = Request()
+        request.journal = journal
+        request.press = press
+        request.repository = None
+        if journal:
+            request.site_type = journal
+        elif repository:
+            request.site_type = repository
+        else:
+            request.site_type = press
+
+        subject = options['subject']
+        to = options['to']
+
+        body_file = os.path.join(settings.BASE_DIR, options['body'])
```

Comment:
This is a bit weird, usually with command line programs you would expect the relative path to be calculated from the current working directory (default behaviour of Python's `open()`)